.drone.yml

```diff
@@ -44,22 +49,27 @@ trigger:
 
 steps:
 - name: build
-  image: grafana/grafana-plugin-ci:1.8.1-alpine
+  image: grafana/grafana-plugin-ci:1.9.5
```

Comment:
Since `1.9.0`, we stopped building alpine-based images. Only debian-based ones are available:

https://hub.docker.com/r/grafana/grafana-plugin-ci/tags